components/brave_rewards/browser/database/publisher_info_database.cc

```diff
@@ -1154,7 +1155,7 @@ void PublisherInfoDatabase::Vacuum() {
 void PublisherInfoDatabase::OnMemoryPressure(
     base::MemoryPressureListener::MemoryPressureLevel memory_pressure_level) {
   DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
-  db_.TrimMemory();
+  TRIM_PUBLISHERS_DB_MEMORY
```

Comment:
why is this a `define` and why would it be defined in `version_info.h`?

Reply:
Removed that change